shared/chat/conversation/messages/attachment.desktop.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, {Component} from 'react'
+import {Avatar, Box, Icon, Text} from '../../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../../styles'
+import * as Constants from '../../../constants/chat'
+
+import type {Props} from './attachment'
+
+const _marginColor = (followState) => ({
```

Comment:
Ditto with other things and if they are shared maybe put them somewhere else (in styles) and import?